go/common/viewingkey/viewing_key.go

```diff
@@ -34,6 +33,7 @@ const (
 	EIP712DomainVersion      = "version"
 	EIP712DomainChainID      = "chainId"
 	EIP712EncryptionToken    = "Encryption Token"
+	EIP712EncryptionTokenV2  = "EncryptionToken"
```

Comment:
Maybe worth expressing this as a list of acceptable token strings?

Reply:
list added